### PR TITLE
Update explorer

### PR DESCRIPTION
**Pull request by andrew-vancil**
Update the data explorer to rely on the new codec data packages

### CODE REVIEW

**Review by cole-brokamp — CHANGES_REQUESTED** [review]

File: inst/codec_shiny/app.R [hunk 1]
```diff
@@ -10,115 +10,168 @@ library(cowplot)
...
+  
+  tracts_sf <- cincy::tract_tigris_2010
+  
+  d_all <-
+    purrr::reduce(dpkgs, dplyr::left_join, by = "census_tract_id_2010", .init = tracts_sf) |>
+    tibble::as_tibble() |>
+    st_as_sf() |>
+    st_transform(st_crs(4326))
+  
+  badges <-
```
[the remark this answers is not in the document]
Comment (andrew-vancil) [reply inst/codec_shiny/app.R:75]: I've got a more programmatic way to build this now, but there's still the same little bit of manual update similar to the top data collection

**Comment by andrew-vancil** [comment]
If this looks good, I'll write a manifest and publish a [dev] version on dev.salt so you can check it out there

**Review by cole-brokamp — APPROVED** [review]

**Comment by andrew-vancil** [comment]
> If this looks good, I'll write a manifest and publish a [dev] version on dev.salt so you can check it out there

Ok, I've got it up on dev.salt as [dev] CoDEC Explorer

**Comment by cole-brokamp** [comment]
let's deploy it on the production version of SALT, but as a development (private) product. we should only be using the dev SALT environment to test new ways to deploy connect on aws.

the buttons don't seem to be scaling with the window size, anyway to make that scale with the rest?

**Comment by andrew-vancil** [comment]
Ok, I've got the app up on connect cloud here: https://connect.posit.cloud/andrew-vancil/content/01923459-ea8e-b78b-cd4f-45b71864e98b

Updates: 
- [x] fixed width on select all and deselect all buttons
- [x] put the univariate switch in the header
- [x] condensed the sidebar to fit better without scrolling (no way to move it to the top of the page)
- [x] added a warning for unselecting all data packages
- [ ] add property code enforcements

**Comment by cole-brokamp** [comment]
posit connect cloud has to use personal namespace

iframe? with pre-load in the background?

**Comment by cole-brokamp** [comment]
Hey @andrew-vancil, I did a lot to reduce the complexity of the code and user interface while still keeping all of the great functionality.  I still want to tweak the layout a little bit to make the selection bars consistent and the options in the card header more consistent.

I also want to think about making the map in the sidebar instead of as a floating window.  I think it would be easier to navigate and use both at once.  We could instead use a switch to view either the plot or the map in the main view versus side-view.  Then, hiding the sidebar will also let the user focus in on one and hide the other if they want.

Here's where I'm at:

image

Can you help to make these changes to get to our next iteration of this PR?

**Pull request merged**